app/src/main/contrib/components/Responsibilities/Overview/List/TemplateLinks.tsx

```diff
@@ -1,17 +1,20 @@
 import * as React from "react";
 import {mainStore} from "../../../../stores/MainStore";
 import {ExtendedResponsibility} from "../../../../models/ResponsibilitySet";
+import { settings } from "../../../../../shared/Settings";
 
 const stochasticParams = require('./stochastic_template_params.csv');
 
 export interface TemplateLinksProps {
     responsibilities: ExtendedResponsibility[];
-    groupId: string
+    groupId: string;
+    touchstoneId?: string;
```

Comment:
I think these `touchstoneId` params should be non-optional, and passed through in both places where `TemplateLinks` is used!